src/editors/utils/formatLibraryImgRequest.ts

```diff
@@ -0,0 +1,134 @@
+import StrictDict from './StrictDict';
+
+/**
+ * A dictionary that maps file extensions to their corresponding MIME types for images.
+ *
+ * @example
+ * acceptedImgMimeTypes.gif // "image/gif"
+ */
+
+const acceptedImgMimeTypes = StrictDict({
+  gif: 'image/gif',
+  jpg: 'image/jpg',
+  jpeg: 'image/jpeg',
+  png: 'image/png',
+  tif: 'image/tiff',
+  tiff: 'image/tiff',
+  ico: 'image/x-icon',
+});
+
+type TinyMCEImageData = {
```

Comment:
Nit: The name `TinyMCEImageData` sounds like this is the data format that tinyMCE requires, but this doesn't seem to be required by TinyMCE (if it was you could import the type like `import type { ImageData } from 'tinymce'`). I can only see this type being used in [FileInfoModalSidebar](https://github.com/openedx/frontend-app-authoring/blob/e118eb5971f0d52854aa5e545b4bf987ec7adc8d/src/files-and-videos/files-page/FileInfoModalSidebar.jsx#L113-L124) and [GalleryCard](https://github.com/openedx/frontend-app-authoring/blob/e118eb5971f0d52854aa5e545b4bf987ec7adc8d/src/editors/sharedComponents/SelectionModal/GalleryCard.jsx#L97-L112). Is there a better name for this interface?

Reply:
May be `GalleryImage`?

Reply:
That sounds better to me :)